tailwind.config.js

```diff
@@ -3,13 +3,13 @@ const plugin = require("tailwindcss/plugin");
 
 /** @type {import("tailwindcss").Config} */
 module.exports = {
-  mode: "jit",
+  // mode: "jit",
```

Comment:
Why was JIT turned off?

Reply:
jit is the default
https://tailwindcss.com/docs/upgrade-guide#migrating-to-the-jit-engine

Reply:
Can we delete the line then?